### PR TITLE
Add built file to /Users/travis.smith/Projects/GO/bin on build

### PR DESCRIPTION
**Pull request by wpsmithtwc**
`go build` allows you to control the output. Why not put the output in the `$GOPATH/bin` folder which will be in a user's PATH (if they have installed GO correctly)?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -74,9 +74,8 @@ If you want to compile it from source, you will need Go 1.7 or later, and the [G
 3. Install dependencies using Glide:  
   `glide install`
 4. Compile the binary:  
-  - Linux/macOS/*nix: `go build -o akamai`
-  - Windows `go build -o akamai.exe`
-5. Move the binary (`akamai` or `akamai.exe`) it to your `PATH`
+  - Linux/macOS/*nix: `go install -o $GOPATH/bin/akamai .``
+  - Windows `go build -o $GOPATH/bin/akamai.exe`
```
Comment (dshafik) [inline README.md]: Would this not be `%GOPATH%` on windows? I don't currently have a Windows box to test on…

File: README.md [hunk 2]
```diff
@@ -76,7 +76,6 @@ If you want to compile it from source, you will need Go 1.7 or later, and the [G
 4. Compile the binary:  
   - Linux/macOS/*nix: `go install -o $GOPATH/bin/akamai .``
   - Windows `go build -o $GOPATH/bin/akamai.exe`
-5. Move the binary (`akamai` or `akamai.exe`) it to your `PATH`
```
Comment (dshafik) [inline README.md]: It might be best to add "5. Ensure that `GOPATH/bin` is in your `PATH`" instead of this?

File: README.md [hunk 3]
```diff
@@ -74,9 +74,8 @@ If you want to compile it from source, you will need Go 1.7 or later, and the [G
 3. Install dependencies using Glide:  
   `glide install`
 4. Compile the binary:  
-  - Linux/macOS/*nix: `go build -o akamai`
-  - Windows `go build -o akamai.exe`
-5. Move the binary (`akamai` or `akamai.exe`) it to your `PATH`
+  - Linux/macOS/*nix: `go install -o $GOPATH/bin/akamai .``
```
Comment (dshafik) [inline README.md]: `go install` doesn't support `-o`, and the step before is `go build`; should we perhaps just combine the steps?

File: README.md (hunk 1 above)
Comment (wpsmithtwc) [inline README.md]: Correct.

**Pull request closed without merging**